app/src/main/java/com/isc/hermes/database/schema/IncidentSchema.java

```diff
@@ -0,0 +1,35 @@
+package com.isc.hermes.database.schema;
+
+/**
+ * This enum represents the fields that the schema of an incident has.
+ */
+public enum IncidentSchema {
+
+    ID("_id"),
+    TYPE("type"),
+    REASON("reason"),
+    DATE_CREATED("dateCreated"),
+    DEATH_DATE("deathDate"),
+    GEOMETRY("geometry"),
+    GEOMETRY_TYPE("geometry.type"),
+    GEOMETRY_COORDINATES("geometry.coordinates");
+
+    private final String field;
+
+    /**
+     * This is the constructor method to initialize the schema field you want to use.
+     * @param field
```

Comment:
One small note about this is that the `@param field` argument is not described in your documentation, consider documenting it.

Reply:
done